tests/test_constrained.py

```diff
@@ -610,5 +610,74 @@ def test_fixed_and_constrained(self):
             self.assertEqual(obj_jac.shape, cons_jac.shape)
             self.assertEqual(obj_jac.shape, (2,))
 
+    def test_fixed_and_constrained_tc(self):
```

Comment:
Is this test in anyway different from `test_fixed_and_constrained`, or is it identical? Cause if so, I think it would be better to add a small for loop to `test_fixed_and_constrained` to run over both minimizers.

Reply:
Similar but different, because trust-constr constraints are entirely different from COBYLA/SLSSQP constraints. Literally 0 overlap.